### PR TITLE
update!: Add Coinsbee to financial services

### PR DESCRIPTION
**Pull request by tacolopo**
Need to add logo to asset, the forked repository doesn't have the assets/img/financial-services folder.

Coinsbee accepts XMR and does not require any verification for orders less than 1000 euros. They accept ephemeral emails and allow VPNs. I have no association to Coinsbee.

Changes proposed in this PR:

- https://discuss.privacyguides.net/t/coinsbee-gift-cards-marketplace/17667

- [x] I have disclosed any relevant conflicts of interest in my post.
- [x] I agree to grant Privacy Guides a perpetual, worldwide, non-exclusive, transferable, royalty-free, irrevocable license with the right to sublicense such rights through multiple tiers of sublicensees, to reproduce, modify, display, perform, relicense, and distribute my contribution as part of this project.
- [x] I am the sole author of this work. 
- [x] I agree to the [Community Code of Conduct](https://www.privacyguides.org/coc).

### CODE REVIEW

File: docs/financial-services.md [hunk 1]
```diff
@@ -84,6 +84,22 @@ These services allow you to purchase gift cards for a variety of merchants onlin
 
 </div>
 
+### Coinsbee
+
+<div class="admonition recommendation" markdown>
+
+![Coinsbee logo](assets/img/financial-services/coincards.svg){ align=right }
+
+**Coinsbee** (available worldwide) allows you to purchase gift cards for 2,500 merchants in 185 countries. No verification is required for orders less than €1,000.
```
Comment (rollsicecream) [inline docs/financial-services.md:93]: Better wording:

```suggestion
**Coinsbee** (available worldwide) allows you to purchase gift cards from 2,500 merchants in 185 countries. No verification is required for orders less than €1,000.
```

File: docs/financial-services.md [hunk 2]
```diff
@@ -84,6 +84,22 @@ These services allow you to purchase gift cards for a variety of merchants onlin
 
 </div>
 
+### Coinsbee
+
+<div class="admonition recommendation" markdown>
+
+![Coinsbee logo](assets/img/financial-services/coincards.svg){ align=right }
```
Comment (rollsicecream) [inline docs/financial-services.md:91]: You are pointing the recommendation card to the CoinCards SVG image, which is a different service recommended on this page....

```suggestion
![Coinsbee logo](assets/img/financial-services/coinsbee.svg){ align=right }
```
The SVG image that you will add should have that filename, of course.

**Review by rollsicecream — CHANGES_REQUESTED** [review]
In addition to what @redoomed1 said....

File: docs/financial-services.md [hunk 3]
```diff
@@ -84,6 +84,22 @@ These services allow you to purchase gift cards for a variety of merchants onlin
 
 </div>
 
+### Coinsbee
+
+<div class="admonition recommendation" markdown>
+
+![Coinsbee logo](assets/img/financial-services/coinsbee.svg){ align=right }
+
+**Coinsbee** (available worldwide) allows you to purchase gift cards from 2,500 merchants in 185 countries. No verification is required for orders less than €1,000.
```
Comment (rollsicecream) [inline docs/financial-services.md:93]: Another better wording: 

```suggestion
**Coinsbee** (available worldwide) allows you to purchase gift cards from 2,500 merchants across 185 countries. No verification is required for orders less than €1,000.
```

**Review by rollsicecream — CHANGES_REQUESTED** [review]

**Comment by tacolopo** [comment]
Thanks to @rollsicecream for starting the forum discussion. My apologies for butchering the process.

**Comment by jonaharagon** [comment]
> Need to add logo to asset, the forked repository doesn't have the assets/img/financial-services folder.

https://github.com/tacolopo/privacyguides.org/tree/main/theme/assets/img/financial-services

**Comment by tacolopo** [comment]
Privacy guides has proved so immensely slow with an inane contribution process that it is not worth contributing to. Nor is it relevant for anyone seeking up-to-date privacy tools.

**Pull request closed without merging**